src/index.html

```diff
@@ -11,6 +11,7 @@
     <link rel="stylesheet" type="text/css" href="/assets/icons/ico.data.svg.css">
     <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,400i,700" rel="stylesheet">
     <link href="https://devinit.org/assets/css/di_website.min.2a5154f5c000.css" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js" type="text/javascript"></script>
```

Comment:
I'm sure you can get your code to function without this dependency ... find a proper alternative that can be installed through `npm`, not JQuery.